### PR TITLE
bpo-25735: math.factorial doc should mention integer return type

### PR DESCRIPTION
**Pull request by akshaysharma096**
Issue link: [math.factorial doc should mention integer return type](https://bugs.python.org/issue25735)

https://bugs.python.org/issue25735

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately our records indicate you have not signed the CLA. For legal reasons we need you to sign this before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

**Comment by akshaysharma096** [comment]
@the-knights-who-say-ni CLA has been signed.

**Comment by chadmiller-amzn** [comment]
The bot isn't that smart yet. File a bug report in the bot's source's github page.

**Comment by chadmiller-amzn** [comment]
Is this necessary? I don't think anyone could realistically think it's not an integer.

File: Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Added documentations for :func:`fabs`, to take only integer input values.
```
Comment (mcscope) [inline Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst]: shouldn't this say "added documentation for func factorial to indicate that returns integer values" ?

Comment (akshaysharma096) [reply Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst]: Noted will do

Comment (akshaysharma096) [reply Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst]: @mcscope you can check again

File: Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst [hunk 2]
```diff
@@ -0,0 +1 @@
+Added documentation for func factorial to indicate that returns integer values
```
Comment (jdemeyer) [inline Misc/NEWS.d/next/Documentation/2018-04-08-19-09-22.bpo-25735.idVQBD.rst:1]: Do we really need a `NEWS` entry for adding 3 words to the doc? I don't think so.

**Review by jdemeyer — CHANGES_REQUESTED** [review]

**Review by csabella — APPROVED** [review]
LGTM

**Comment by csabella** [comment]
@akshaysharma096, thank you for the contribution.  It has now been merged!  :tada:

**Pull request merged**